Goal App/Controllers/PersonalGoalsViewController.swift

```diff
@@ -27,7 +27,10 @@ class PersonalGoalsViewController: UIViewController {
     }
 
     override func viewWillAppear(_: Bool) {
-        getGoalsList(segment: selectedSegmentIndex)
+        Task {
+            originalGoalsList = await firestoreUtil.getGoals()
+            getGoalsList(segment: selectedSegmentIndex)
```

Comment:
`getGoalsList` can be renamed to `getGoals` what's the use of segment parameter? The intent of that is not clear